File: fifth-exercises/python/9.py
valor_casa = float(input("Digite o valor da casa a comprar: R$ "))
salario = float(input("Digite o salário: R$ "))
anos = int(input("Digite a quantidade de anos a pagar: "))

meses = anos * 12  # Converter anos em meses
prestacao = valor_casa / meses

limite_prestacao = salario * 0.30  # 30% do salário

if prestacao <= limite_prestacao:
    print("Empréstimo aprovado! Prestação mensal: R$", prestacao)
else:
    print("Empréstimo não aprovado. Prestação mensal excede 30% do salário.")

File: fifth-exercises/c/9.c
#include <stdio.h>

int main() {
    float valorCasa, salario, prestacao;
    int anos;

    printf("Digite o valor da casa a comprar: R$ ");
    scanf("%f", &valorCasa);

    printf("Digite o salário do comprador: R$ ");
    scanf("%f", &salario);

    printf("Digite a quantidade de anos a pagar: ");
    scanf("%d", &anos);

    int meses = anos * 12; // Converter anos em meses

    prestacao = valorCasa / meses;

    if (prestacao > salario * 0.3) {
        printf("Empréstimo não aprovado. Prestação mensal excede 30%% do salário.\n");
    } else {
        printf("Empréstimo aprovado! Prestação mensal: R$ %.2f\n", prestacao);
    }

    return 0;
}
